joint_limits:
  joint1:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 2.0
  joint2:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 2.0
  joint3:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 2.0
  joint4:
    has_velocity_limits: true
    max_velocity: 1.5
    has_acceleration_limits: true
    max_acceleration: 2.0
  joint5:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 3.0
  r_joint:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 3.0
